tfhe/src/core_crypto/algorithms/mod.rs

```diff
@@ -44,7 +44,7 @@ pub mod seeded_lwe_public_key_decompression;
 pub mod slice_algorithms;
 
 #[cfg(test)]
-mod test;
+pub mod test;
```

Comment:
why is that needed ?

Reply:
To get access to the keys generation function from outside the algorithms/ directory. I need the functions in the keycache thus outside of this very module.

Reply:
hmm, ah for the fft impl stuff ? pub(crate) should suffice

Reply:
No it for pre-generation, so it's in core_crypto/keycache.rs but in anycase pub(crate) seems to work.

Reply:
can't the test be in core_crypto::algorithms::tests to make it easier ?